functors_task_1/student/include/impl.h

```diff
@@ -16,16 +29,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
```

Comment:
please mark "in" function params const by default, as modifying them is considered as a bad practice